### PR TITLE
chore: move library settings cron expression to setting select element

### PR DESCRIPTION
**Pull request by zackpollard**

### CODE REVIEW

**Comment by mertalev** [comment]
What's the motivation of this change? I think you should still be able to set the cron to something besides one of the presets.

**Comment by zackpollard** [comment]
> What's the motivation of this change? I think you should still be able to set the cron to something besides one of the presets.
> 
> Oh, never mind. I thought this was removing the text field.

I was in the middle of replying :laughing:

**Review by mertalev — APPROVED** [review]

File: web/src/lib/components/admin-page/settings/library-settings/library-settings.svelte [hunk 1]
```diff
@@ -60,25 +61,13 @@
...
-                bind:value={config.library.scan.cronExpression}
-              >
-                {#each cronExpressionOptions as { title, expression }}
-                  <option value={expression}>{title}</option>
-                {/each}
-              </select>
-            </div>
+            <SettingSelect
+              options={cronExpressionOptions}
+              disabled={disabled || !config.library.scan.cronExpression}
```
Comment (mertalev) [inline web/src/lib/components/admin-page/settings/library-settings/library-settings.svelte:66]: Is it intentional that you changed this from `enabled` to `cronExpression`?

Comment (zackpollard) [reply web/src/lib/components/admin-page/settings/library-settings/library-settings.svelte]: Oops, copy paste error :sweat_smile: Fixed

**Pull request merged**